node.gyp

```diff
@@ -207,7 +214,9 @@
         'src/udp_wrap.cc',
         'src/util.cc',
         'src/uv.cc',
+        'src/loader/module_wrap.cc',
         # headers to make for a more pleasant IDE experience
+        'src/loader/module_wrap.h',
```

Comment:
I’m also wondering, is there any particular reason for having this be in its own directory?